### DISCUSSION TITLE
When running the packaged program, it prompts importlib.metadata.PackageNotFoundError: pyrfc

### DISCUSSION THREAD

**Discussion by jiangtaobian** [pyinstaller/pyinstaller#8566] [PyInstaller] [answered]
When running the packaged program, it prompts:
“importlib.metadata.PackageNotFoundError: pyrfc”
The code uses the third-party module pyrfc, and has configured the windows environment parameter PATH and the environment variable of the SAPNWRFC_HOME directory. It can run normally in the development environment, but an error will be reported after running.
Please tell me how to deal with it.
![微信截图_20240527135258](https://github.com/pyinstaller/pyinstaller/assets/156577903/3ef58709-6ebd-4b15-bfbd-fd23f6cec5a0)

**Comment by rokm** [comment, marked as answer]
You need to ensure that metadata for `pyrfc` is collected (it is not by default); i.e., try adding [`--copy-metadata pyrfc`](https://pyinstaller.org/en/stable/usage.html#cmdoption-copy-metadata) to your PyInstaller command.
